lib/cloud_controller/config_schemas/api_schema.rb

```diff
@@ -297,6 +297,7 @@ class ApiSchema < VCAP::Config
             optional(:port) => Integer,
             timeout_in_milliseconds: Integer,
             ca_cert_path: String,
+            optional(:query_raise_on_mismatch) => bool,
```

Comment:
This code is not actually publicly modifiable.

Because the v3 integration tests spin up a new cloud_controller in a separate process, there's no way to monkey patch the cloud_controller code before it gets spun up, i.e. to enable the RaiseOnMismatch property. To get around that, this is a hidden cloud controller configuration that we override when we create our `cloud_controller.yml` configuration file in v3 integration tests.